frontend/src/util/config.ts

```diff
@@ -4,8 +4,8 @@ import dotenv from "dotenv"
  * above line is causing errors, needs to be resolved to use .env
  */
 //const server = process.env.EXTERNAL_IP;
-//const server = "localhost:5000";
-const server = '34.82.220.171';
+const server = "localhost:5000";
+//const server = '34.82.220.171';
```

Comment:
I know it's a hassle but we need this to stay on the IP and not localhost for merge into development

Reply:
marking as needs work just in case